portal/views.py

```diff
@@ -34,20 +34,6 @@ def all_hosts(request):
 def ansible_hosts(request):
     """Generates Ansible dynamic inventory format"""
 
-    # hint OS by assuming all routers are Mikrotik
-    os = {
-        'sector': 'mikrotik',
-        'ptp': 'mikrotik',
-        'edgerouter': 'mikrotik',
-        'cellrouter': 'mikrotik',
-        'client': 'mikrotik',
-        'server': 'linux',
-        'anycast': None,
-        'pdu': None,
-        'kvm': None,
-        'other': 'linux',
-    }
```

Comment:
Please make this backwards compatible so that we still have a way to target all the Mikrotik devices. Maybe preserve this default OS identification for when OS is NULL.

Reply:
This will be handled in a post-migration SQL action.  'routeros' will be set for all the 'sector', 'ptp', 'edgerouter', 'cellrouter', 'client' hosts.

Reply:
This might be what you meant by "post-migration", but a manually-written line near the end of the generated migration could handle this.

Reply:
Intended SQL:

UPDATE portal_host SET os='routeros' WHERE type IN ('sector', 'ptp', 'edgerouter', 'cellrouter', 'client');